services/addons/images/firmware_manager/commsignia_upgrader.py

```diff
@@ -60,6 +69,47 @@ def upgrade(self):
             self.cleanup()
             self.notify_firmware_manager(success=False)
 
+    def post_upgrade(self):
+        if self.wait_until_online() == -1:
+            raise Exception("RSU offline for too long after firmware upgrade")
+        try:
+            time.sleep(60)

```

Comment:
On the previous call we wait for 3 minutes for the RSU to come back online, why the additional minute wait before executing script?

Reply:
This is to allow for the RSU to become available for SSH since a device will respond to a ping before allowing it to SSH.

Reply:
That makes sense, I was wondering if it had to do with the device saying it's ready before it's apps are ready to do anything.